test_board.py

```diff
@@ -23,48 +32,123 @@ def test_when_a_user_picks_a_spot_board_is_updated_with_their_spot():
     player1 = 'X'
     spot = 1
     game_board.new_game()
-    update_state(game_board.board_state, player1, spot)
-    assert game_board.board_state == ["X", " ", " ", " ", " ", " ", " ", " ", " "]
+    BoardState.update_state(game_board.board_state, player1, spot)
+    assert game_board.board_state == [
+                                        "X", "", "", 
+                                        "", "", "", 
+                                        "", "", ""
+                                     ]
 
 def test_when_a_user_picks_a_spot_smaller_then_exists_spot_exists_is_False():
     game_board = Board()
     spot = -1
     game_board.new_game()
-    assert spot_exists(game_board.board_state, spot) == False
+    assert SpotStates.spot_exists(game_board.board_state, spot) == False
 
 def test_when_a_user_picks_a_spot_larger_then_exists_spot_exists_is_False():
     game_board = Board()
     game_board.new_game()
     spot = len(game_board.board_state) + 1 
-    assert spot_exists(game_board.board_state, spot) == False
+    assert SpotStates.spot_exists(game_board.board_state, spot) == False
 
 def test_when_a_user_picks_a_spot_that_exists_spot_exists_is_True():
     game_board = Board()
     game_board.new_game()
     spot = 1 
-    assert spot_exists(game_board.board_state, spot) == True
+    assert SpotStates.spot_exists(game_board.board_state, spot) == True
     
 def test_when_a_user_picks_an_occupied_spot_is_occupied_will_be_True():
     game_board = Board()
     spot = 1
     player1 = 'X'
     game_board.new_game()
-    update_state(game_board.board_state, player1, spot)
-    assert is_occupied(game_board.board_state, spot) == True
+    BoardState.update_state(game_board.board_state, player1, spot)
+    assert SpotStates.is_occupied(game_board.board_state, spot) == True
 
 def test_game_ends_when_board_is_full():
     game_board = Board()
-    game_board.board_state = ["X","O","X","O","X","O","X","O","O"]
-    assert is_draw(game_board.board_state) == True
+    game_board.board_state = [
+                                "X","O","X",
+                                "O","X","O",
+                                "X","O","O"
+                             ]
+    assert EndStates.is_draw(game_board.board_state) == True
 
 def test_game_continues_if_board_is_not_full():
```

Comment:
this test is a bit confusing since it's looking to see if the game continues, but the assertion is about whether or not the game is a draw.

Reply:
Good point! I'll have to change the test to check that a draw actually ends a game.